Core/ContentBlockerRulesManager.swift

```diff
@@ -48,6 +56,12 @@ public class ContentBlockerRulesManager {
 
     /// Return compiled rules for the current content blocking configuration.  This may return a precompiled rule set.
     public func compiledRules(completion: ((WKContentRuleList?) -> Void)?) {
+        guard !isCompilingRules else {
```

Comment:
One thing I'm worried about here is this scenario:

1. App launches for the very first time, and starts compiling the built-in rules
2. App downloads the new tracker data
3. App tries to recompile the freshly downloaded data, but the first compilation is still running, so it hits the early return.

@brindy Not sure about the best way to solve for that, what do you think?